addons/evm/src/commands/actions/call_contract.rs

```diff
@@ -252,21 +251,23 @@ impl CommandImplementation for SignEVMContractCall {
                             "".into(),
                             Some(format!("Check account nonce")),
                             ActionItemStatus::Todo,
-                            ActionItemRequestType::ReviewInput(ReviewInputRequest {
-                                input_name: "".into(),
-                                value: Value::integer(transaction.nonce().into()),
-                            }),
+                            ReviewInputRequest::new(
+                                "",
+                                &Value::integer(transaction.nonce().into()),
+                            )
+                            .to_action_type(),
                             ACTION_ITEM_CHECK_NONCE,
                         ),
                         ActionItemRequest::new(
                             &Some(construct_did.clone()),
                             "µSTX".into(),
                             Some(format!("Check transaction fee")),
                             ActionItemStatus::Todo,
-                            ActionItemRequestType::ReviewInput(ReviewInputRequest {
-                                input_name: "".into(),
-                                value: Value::integer(transaction.gas_limit().try_into().unwrap()), // todo
-                            }),
+                            ReviewInputRequest::new(
+                                "",
```

Comment:
This field is often empty, is it indicating that it should be an optional? that way we'd have a clean way to get rid of the `()` in the supervisor UI (?)

Reply:
It probably is optional, or not required at all. I have a ticket to look into removing some specific code that is related to this, it might enable removing this field entirely.

But since this PR was intended to get evm web_wallet working, I didn't want to change more than necessary. I'll add a note to my ticket to look into removing this field as well.